dpe/src/x509.rs

```diff
@@ -895,6 +930,51 @@ impl X509CertWriter<'_> {
         Ok(bytes_written)
     }
 
+    /// Encode ExtendedKeyUsage extension
```

Comment:
Could you update the comment to explain which EKU OID will be included (based on whether cert is_ca)

Reply:
Done